tbdex/rfq/signature_verifier.go

```diff
@@ -0,0 +1,10 @@
+package rfq
+
+import (
+	"github.com/TBD54566975/tbdex-go/tbdex"
+	"github.com/tbd54566975/web5-go/jws"
+)
+
+var VerifySignatureFunc = func(r *RFQ, signature string) (*jws.Decoded, error) {
```

Comment:
what's the purpose of extracting this?

Reply:
i was using it so i can hotswap the behavior i want this func to return during test. but it is not needed anymore so i removed it